src/NuGet.Core/NuGet.CommandLine.XPlat/ExitCodes.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.CommandLine.XPlat
+{
+    internal static class ExitCodes
+    {
+        // Exit code use when the command line arguments are parsed successfully and the command is run successfully
+        internal const int EXIT_SUCCESS = 0;
+
+        // Exit code used when te command line arguments are not parsed successfully.
+        internal const int EXIT_COMMANDLINE_ARGUMENT_PARSING_FAILURE = 1;
```

Comment:
There are no tests in this PR that validate that the bug reported is actually fixed (and therefore doesn't regress again in the future).

I don't think it will be difficult to write a `Dotnet.Integration.Test` that captures the process output and validates that the result is valid JSON.  I strongly recommend that when it comes to bug fixes, try to practise [test driven development](https://en.wikipedia.org/wiki/Test-driven_development) when possible, and write a test that fails first (to ensure the test actually works), then fix the product code to make the test pass. Some things are too hard to test, and TDD is a pain for new features, but for "little" bugs like this, I think it's quite important.